tests/test_api.rs

```diff
@@ -242,3 +243,34 @@ fn exception() {
   });
   isolate.exit();
 }
+
+#[test]
+fn object() {
+  setup();
+  let mut params = v8::Isolate::create_params();
+  params.set_array_buffer_allocator(
+    v8::array_buffer::Allocator::new_default_allocator(),
+  );
+  let mut isolate = v8::Isolate::new(params);
+  let mut locker = v8::Locker::new(&mut isolate);
+  v8::HandleScope::enter(&mut locker, |scope| {
+    let mut context = v8::Context::new(scope);
+    context.enter();
+    let null = new_null(scope);
+    // TODO: safer casts.
+    let null: v8::Local<v8::Value> =
+        unsafe { std::mem::transmute_copy(&null) };
+
+    // TODO: safer casts.
+    let n1: Local<v8::Name> = unsafe { std::mem::transmute_copy(&v8::String::new(scope, "a", v8::NewStringType::Normal).unwrap()) };
+    let n2: Local<v8::Name> = unsafe { std::mem::transmute_copy(&v8::String::new(scope, "b", v8::NewStringType::Normal).unwrap()) };
+    let names = vec![n1, n2];
+    // TODO: safer casts.
+    let v1: v8::Local<v8::Value> = unsafe { std::mem::transmute_copy(&v8::Number::new(scope, 1.0)) };
+    let v2: v8::Local<v8::Value> = unsafe { std::mem::transmute_copy(&v8::Number::new(scope, 2.0)) };
+    let values = vec![v1, v2];
```

Comment:
This is horrible, we need to figure out #51  quickly

Reply:
Can you pull the constructors out of the unsafe block?

Also rather than `// TODO: safer casts` put
```
// TODO Safer casts https://github.com/denoland/rusty_v8/issues/51
```

Reply:
added `cast<U, T>(Local<T>) -> Local<U>` utility function